packages/ag-charts-enterprise/src/main-modules.ts

```diff
@@ -1,9 +1,15 @@
 import { AgCharts, VERSION, _Scene, _Theme, _Util, setupCommunityModules } from 'ag-charts-community';
 import type { IntegratedModule } from 'ag-charts-types';
 
+import { LicenseManager as RealLicenseManager } from './license/licenseManager';
 import { setupEnterpriseModules as internalSetup } from './setup';
 
-export { AgCharts, VERSION };
+const LicenseManager = {
+    setLicenseKey(key: string) {
+        RealLicenseManager.setLicenseKey(key);
+    },
+};
+export { AgCharts, VERSION, LicenseManager };
```

Comment:
why aren't we exporting the `RealLicenseManager`?

Reply:
Just from the perspective of only exporting the absolutely necessary API, `LicenseManager` has a larger API surface than users need to consider, so this 1) simplifies the exported API (from a user perspective) and 2) avoids users being able to do unexpected things with the `LicenseManager`.